confmap/internal/e2e/types_test.go

```diff
@@ -634,3 +634,52 @@ logging:
 		cfgStr.Field,
 	)
 }
+
+func TestIssue10799(t *testing.T) {
```

Comment:
Can we give it a more descriptive name, like TestEnvVarInPipeline etc.?

Same for the file names

Reply:
might be worth adding a link to the issue in a comment in the test

Reply:
Done!